Got an MVP for jobs using the scraper.  Returns up to 15 jobs for a j…

DrewRust opened the pull request:
…ob type, city, and state.  I have to redo the csv file and then push it to the db or have Julia help me with that.  I would ideally like to query the db and should be able to once I get a better collections of jobs for all the states. Thanks guys!

DrewRust committed:
Got an MVP for jobs using the scraper.  Returns up to 15 jobs for a job type, city, and state.

tigju commented:
Andrew, you did a good job doing both with a scraper and db. Your code to retrieve data from db works, you just need to import your function which you've created in db.py to ml.py (from app.db import select_available_jobs). Also, in the select_available_jobs function you are retrieving only one column, so you don't need a star (*) in the query. You need to change .select(*columns) to .select(columns). The other thing is a table that you are retrieving data from doesn't have some cities (New York, Los Angeles), so the result is empty. But it's ok as long as it doesn't throw an error.

DrewRust committed:
Got an MVP for jobs using the scraper. Just updating my changes with main so we can merge.

tigju reviewed (dismissed):
Scraper is running smoothly, great job!

jckaizen committed:
Cleaned up code

jckaizen committed:
Fixed code typo

Line 495 'weather_conditions' was changed to weather_temperature. I think that was a typo or a mistake unless I'm wrong but I don't any reason for the change so I reverted it back to its original.

jckaizen committed:
Cleaned up the code

jckaizen commented on app/ml.py:
```diff
@@ -492,4 +492,43 @@ async def get_weather_conditions(city: City):
     
-    return {'City': city.city, 'State': city.state, 'weather_conditions': weather_cond}
\ No newline at end of file
+    return {'City': city.city, 'State': city.state, 'weather_conditions': weather_cond}
```
I changed weather_temperature back to the original which was weather_conditions. I didn't see any reason for the change plus it was going to mess up the function here. I'm assuming a typo?

tigju replied:
Julio, nice catch!  It is definitely a typo. Good job fixing it!

DrewRust replied:
You guys are amazing!

julioe27 commented on app/ml.py:
```diff
@@ -492,4 +492,43 @@ async def get_weather_conditions(city: City):
     
-    return {'City': city.city, 'State': city.state, 'weather_conditions': weather_cond}
\ No newline at end of file
+    return {'City': city.city, 'State': city.state, 'weather_conditions': weather_cond}
```
I changed weather_temperature back to weather_conditions (which was the original). I didn't see a reason to change change this to temperature plus it would have throw an error when using this function. I'm assuming a typo?

jckaizen reviewed (approved):
Good Job @DrewRust with the scrapper! I cleaned up your code a little bit. Also there seemed to be a typo that would have messed up one of the weather functions in ml.py but I changed that. Again, good job!

tigju reviewed (approved):
Good job Andrew , scraper works perfectly. 
Thanks Julio for finding a typo, I guess it happened when Andrew was resolving  conflicts with main branch. Nice job guys, team work!
